### ISSUE TITLE
Training loss 

### ISSUE THREAD

**Issue by RaymondCover** [visionml/pytracking#259]
![image](https://user-images.githubusercontent.com/48719188/121772181-6f76d300-cba6-11eb-978c-7efadf2a9c12.png)
When I use superdimp settings, I found the bb_ce will become negative values, Do you know what caused this? Thanks.

**Comment by nwpuwy** [comment]
I have met same problem,how do you solve it?

**Comment by ChuzzZz** [comment]
I met same problem, how do you solve it?

**Comment by 2006pmach** [comment]
This is not a problem but rather the expected behaviour for the employed loss function for probabilistic bounding box regression. You can find the details in the [paper](https://openaccess.thecvf.com/content_CVPR_2020/papers/Danelljan_Probabilistic_Regression_for_Visual_Tracking_CVPR_2020_paper.pdf). I think Eq. 8 is the reason for a negative loss since the constant term is omitted from here on (@martin-danelljan feel free to correct if needed).

**Comment by ChuzzZz** [comment]
> This is not a problem but rather the expected behaviour for the employed loss function for probabilistic bounding box regression. You can find the details in the [paper](https://openaccess.thecvf.com/content_CVPR_2020/papers/Danelljan_Probabilistic_Regression_for_Visual_Tracking_CVPR_2020_paper.pdf). I think Eq. 8 is the reason for a negative loss since the constant term is omitted from here on (@martin-danelljan feel free to correct if needed).

Thanks for your reply!
